commit 5655a0b58e1ba27b05c94c684c74b7078cb814d5
Author: Adam Conrad <adconrad@0c3.net>
Date:   Tue Jan 22 23:05:33 2013 -0700

    Build elf/tst-array2 and elf/tst-array5 with --no-as-needed
    
    	* elf/Makefile (LDFLAGS-tst-array2): Add $(no-as-needed).
    	(LDFLAGS-tst-array5): Likewise.

diff --git a/elf/Makefile b/elf/Makefile
index 84f8397..6d4b365 100644
--- a/elf/Makefile
+++ b/elf/Makefile
@@ -849,6 +849,9 @@ LDFLAGS-tst-execstack-needed = -Wl,-z,noexecstack
 LDFLAGS-tst-execstack-prog = -Wl,-z,execstack
 endif
 
+LDFLAGS-tst-array2 = $(no-as-needed)
+LDFLAGS-tst-array5 = $(no-as-needed)
+
 $(objpfx)tst-array1.out: tst-array1.exp $(objpfx)tst-array1
 	$(built-program-cmd) > $@
 	cmp $@ tst-array1.exp > /dev/null
